### PR TITLE
Document base Windows Server 2016 template

### PR DESCRIPTION
**Pull request by sodar**

### CODE REVIEW

**Comment by sodar** [comment]
recheck no bug

File: ansible/Windows-Server-2016-Basic-Template.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- Password policy change
+
+    ```powershell
+    Set-LocalUser -Name Administrator -PasswordNeverExpires:$true
+    ```
+
+- Disable firewall
+
+    ```powershell
+    Set-NetFirewallProfile | Set-NetFirewallProfile -Enabled False
```
Comment (jablonskim) [inline ansible/Windows-Server-2016-Basic-Template.md]: Please change to `Get-NetFirewallProfile`

File: ansible/Windows-Server-2016-Basic-Template.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    .\ConfigureRemotingForAnsible.ps1 -DisableBasicAuth -EnableCredSSP -ForceNewSSLCert -SkipNetworkProfileCheck
+    ```
+
+- Cleanup
+
+    ```powershell
+    Dism /Online /Cleanup-Image /StartComponentCleanup
+    ```
+
+- Convert to template
```
Comment (jablonskim) [inline ansible/Windows-Server-2016-Basic-Template.md]: Maybe add 'Shutdown VM' step here.

File: ansible/Windows-Server-2016-Basic-Template.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# Windows Server 2016 - Basic template installation
+
+### VM hardware
+
+- 1 vCPU, hardware virtualization enabled
```
Comment (jablonskim) [inline ansible/Windows-Server-2016-Basic-Template.md]: Could we change all lists to numbered lists?

**Comment by sodar** [comment]
recheck no bug

**Comment by sodar** [comment]
recheck no bug

**Pull request merged**